llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -344,7 +349,8 @@ static void addReplicateRegions(VPlan &Plan) {
            vp_depth_first_deep(Plan.getEntry()))) {
     for (VPRecipeBase &R : *VPBB)
       if (auto *RepR = dyn_cast<VPReplicateRecipe>(&R)) {
-        if (RepR->isPredicated())
+        // TODO: Support for multiple-def replications.
+        if (RepR->isPredicated() && RepR->getNumDefinedValues() == 1)
```

Comment:
`createReplicateRegion()` also does not support multiple defs (unclear how to handle the `VPPredInstPHIRecipe`).